### PR TITLE
DEV-4539-piscine-js-issue-with-fifty-shades-of-cold-exercice

### PR DESCRIPTION
**Pull request by nprimo**
- change querySelector for first test

### CODE REVIEW

File: dom/fifty-shades-of-cold_test.js [hunk 1]
```diff
@@ -13,10 +13,13 @@ export const tests = []
...
-  const style = await page.$$eval('style', nodes => nodes[1].innerHTML)
-  const classes = style
-    .split('}')
-    .map(s => s.replace(/(\.|{|:|;|\s+)/g, ''))
+  const style = await page.$$eval('head style', nodes =>
+    nodes.map(n => n.innerText),
+  )
+
+  const classes = style[0]
+    .split('\n')
```
Comment (mikysett) [inline dom/fifty-shades-of-cold_test.js:21]: This line will make valid solutions fail the test since start a new line for every new element is a best practice but has no semantic value in CSS.

It could be replaced by `.split('}')`

**Review by mikysett — CHANGES_REQUESTED** [review]
A small modification in order to ensure valid solutions do not fail tests.

It must be said there is room for improvement since many invalid solutions may also pass the tests.
This is due to excessive usage of regex and to splitting technique to separate the different classes. Those edge cases require a more profound refactor and can be done at a later point in time.

**Review by mikysett — APPROVED** [review]
LGTM
